sklearn/cluster/hierarchical.py

```diff
@@ -595,6 +609,11 @@ class AgglomerativeClustering(BaseEstimator, ClusterMixin):
     ----------
     n_clusters : int, default=2
         The number of clusters to find.
+        NOTE: You should set either n_cluster or distance_threshold, NOT both.
```

Comment:
variables in docstrings of public classes or functions should be surrounded by double back-ticks. 
i.e: \`\`n_clusters\`\` to render as ``n_clusters``